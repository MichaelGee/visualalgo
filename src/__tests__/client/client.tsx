import App from '@app/client/app';
import * as React from 'react';
import { cleanup, render } from 'react-testing-library';

afterEach(cleanup);

test('Client Side Render', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});

test('Makes request when button is clicked', () => {
    
});
