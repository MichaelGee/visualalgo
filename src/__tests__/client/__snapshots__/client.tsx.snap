// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client Side Render 1`] = `
<DocumentFragment>
  <h1>
    Sieve of Eratosthenes Demo
  </h1>
</DocumentFragment>
`;
